src/main/java/org/mockito/internal/configuration/GlobalConfiguration.java

```diff
@@ -25,6 +26,10 @@
     IMockitoConfiguration getIt() {
         return GLOBAL_CONFIGURATION.get();
     }
+    //back door for testing
+    public static void removeIt() {
+        GLOBAL_CONFIGURATION.remove();
+    }
```

Comment:
Don't add this method here, for unit tests there's this class `org.mockito.configuration.MockitoConfiguration` in the test sources